tests/post/features/monitoring.feature

```diff
@@ -42,4 +42,10 @@ Feature: Monitoring is up and running
     Scenario: Ensure deployed Prometheus rules match the default
         Given the Kubernetes API is available
         And we have 1 running pod labeled 'prometheus=prometheus-operator-prometheus' in namespace 'metalk8s-monitoring'
-        Then the deployed Prometheus alert rules are the same as the default alert rules
\ No newline at end of file
+        Then the deployed Prometheus alert rules are the same as the default alert rules
+
+    Scenario: Volume metrics can be found based on device name
+        Given the Kubernetes API is available
+        And I have created a test Volume
+        And the Prometheus API is available
+        Then I can get I/O stats for the test Volume device
```

Comment:
To make the test faster/less flaky, I guess you could skip the creation of a brand new volume and try to get the stats from one of the two already existing volume (alertmanager's one or prometheus' one).

Reply:
Right... But I didn't want to rely on the fact that this volume _may_ be using a loop device (ideally, our `post` E2E tests are non intrusive, so we can run them to perform a sanity check against a "real" cluster).

Reply:
ha yeah right, we want to check against a loop device specifically (since attaching a real partitioned volume would be even flakier…)